styles.css

```diff
@@ -0,0 +1,428 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  margin:0;
+  font-family: sans-serif;
+  font-size: 18px;
```

Comment:
As you will learn later on on the course we are going to be getting away from using px since they are not responsive. Consider using % or em/rem instead.